apps/skilavottord/ws/src/app/api.graphql

```diff
@@ -2,22 +2,61 @@
 # THIS FILE WAS AUTOMATICALLY GENERATED (DO NOT MODIFY)
 # ------------------------------------------------------
 
-type Car {
+type RecyclingPartner {
   id: ID!
   name: String!
-  model: String!
+  address: String!
+  postNumber: Float!
+  website: String
+  phone: String
+  active: Boolean
+}
+
+type Car {
+  permno: ID!
+  type: String!
   color: String!
-  year: String!
+  newregdate: String!
   recyclable: Boolean!
+  recyclingPartner: RecyclingPartner
+}
+
+type Carowner {
+  nationalId: ID!
+  name: String!
+  mobile: String
+  cars: [Car!]
 }
 
 type Carowner {
   nationalId: ID!
   name: String!
   mobile: String
+<<<<<<< HEAD
```

Comment:
It seems you have done some merge at some point which did not go well and now you need to clean it up before it is usable again.